.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -41,9 +41,8 @@ like this
 <!--- Go over all the following points, and put an `x` in all the boxes that apply. -->
 <!--- If you're unsure about any of these, don't hesitate to ask. We're here to help! -->
 <!--- The entire development process is outlined here: https://docs.vyos.io/en/latest/contributing/development.html -->
-- [ ] I have read the [**CONTRIBUTING**](https://github.com/vyos/vyos-1x/blob/current/CONTRIBUTING.md) document
+- [ ] I have read the [**CONTRIBUTING**](https://github.com/vyos/vyos-1x/blob/sagitta/CONTRIBUTING.md) document
```

Comment:
You merge this into equuleus branch - but I guess the CONTRIBUTION guidelines should always refer to the latest (current) ones.

Reply:
Ok, Noted, will fix this.

Reply:
Fixed this in other PR